### ISSUE TITLE
Error in compiling on Windows

### ISSUE THREAD

**Issue by trungpham2606** [timy90022/One-Shot-Object-Detection#1]
@timy90022 
Hello, tks for sharing a great work.
Iam in the very first steps to run your model. But Iam stuck in compiling part.
Iam having this error: 
![image](https://user-images.githubusercontent.com/35657426/70031505-f9230900-15dd-11ea-8502-eb163ae3136f.png)

Iam using windows 10, cuda 9.0, cudnn 7.0.5, pytorch 1.0.1

**Comment by trungpham2606** [comment]
@timy90022 In addition, the pretrained weights file you uploaded are wrong. They are exactly the same as reference images file. Can you check them again ?

**Comment by timy90022** [comment]
Hi trungpham2606, 

Can you give me more information about pertained weights and reference images file?

**Comment by trungpham2606** [comment]
@timy90022 i was following your instructions:
for the reference images file:
![image](https://user-images.githubusercontent.com/35657426/70104780-b6127580-1671-11ea-8e7b-0db7b1628a65.png)

and model weights for testing:
![image](https://user-images.githubusercontent.com/35657426/70104799-c4f92800-1671-11ea-9a25-fbeac19a7709.png)

But when I accessed those links, the files are the same.

**Comment by timy90022** [comment]
Thanks for reminding me.
I updated the link. You can try it again.

**Comment by trungpham2606** [comment]
@timy90022 Oke, I see. I will try it right now.

**Comment by trungpham2606** [comment]
@timy90022 
Iam having this error while loading the pretrained weights
![image](https://user-images.githubusercontent.com/35657426/70106175-79e11400-1675-11ea-812d-49b7eb6641f3.png)

**Comment by timy90022** [comment]
Can you give me more information about your problem?
And, what is check_net.py ?

**Comment by trungpham2606** [comment]
@timy90022 check_net is actually your test_net. I have to rename it due to the discomfort of Pycharm when the file name has word 'test' in it. 

Config file Iam using is res50_1.yml
I had successfully loaded the dataset for testing.
But when loading the pretrained weights from:  r'pretrained/res50/coco\faster_rcnn_1_10_1663.pth'

**Comment by timy90022** [comment]
You load the wrong file. You need to download the checkpoint file in [link](https://drive.google.com/file/d/1FV7TpTSgF0pwGxshqUSK-AvhXHSAObo4/view).

And, save in 'models/res50/coco\faster_rcnn_1_10_1663.pth'

It is different between checkpoint file and pretrained weights file.

**Comment by trungpham2606** [comment]
@timy90022 it's just about the file directory. I can change it manually ^^. The main problem is that loading config file is not correct. I can now load the model successfully. 
But the image was shown is so weird 
![image](https://user-images.githubusercontent.com/35657426/70108169-338eb380-167b-11ea-81c3-63bf6a9b96a9.png)

**Comment by howardyclo** [comment]
@trungpham2606 We assume that our codebase should be run on Linux-like system since we develop our codebase based on Ubuntu. Therefore we recommend you to run your code on Linux-like system in order to prevent unexpected errors. Thanks.

**Comment by trungpham2606** [comment]
@howardyclo thanks for your prompt response. 
I have 1 more question about the part in test_net.py
![image](https://user-images.githubusercontent.com/35657426/70109836-b74a9f00-167f-11ea-95bc-2d7ff86e7b0f.png)

When showing the o_query image here, it not RGB image. I think you just convert the image back to RGB but its value not in range (0, 255).

**Comment by howardyclo** [comment]
@trungpham2606 How about trying replacing the line 379  `cv2.imwrite('./test_img/%d_d.png'%(i), im2show)` with your `cv2.imshow`?

**Comment by trungpham2606** [comment]
@howardyclo Yeah, the image shown above is the result when I replace cv2.imwrite by cv2.imshow

**Comment by howardyclo** [comment]
@trungpham2606 But you place `cv2.imshow` before un-normalizing the `o_query` (query image), which is not correct I think? The orginal code `cv2.imwrite` is located after un-normalizing `o_query` and concatenating with `im2show`. Please check again. Thanks.

**Comment by trungpham2606** [comment]
@howardyclo yeah I know, I just want to show the o_query image before normalizing as well to see its range. 
![image](https://user-images.githubusercontent.com/35657426/70111016-908e6780-1683-11ea-8097-cb328c64dd10.png)

**Comment by howardyclo** [comment]
@howardyclo We tested this on Ubuntu and it can correctly show images, so we are not sure the reason why you cannot correctly show the image here. Can you show us the shape and the range of `im2show`? You can simply check its min and max values to show the range.

**Comment by trungpham2606** [comment]
@howardyclo  here it is
![image](https://user-images.githubusercontent.com/35657426/70112227-3ee7dc00-1687-11ea-9896-aa21a3c0de59.png)

Can you show me your image ?
Oh, it just about the version of opencv I guess, when i save the image, it can display properly, sry for wasting your time ^^
here is the result for the first image from coco 2014 valid set
![image](https://user-images.githubusercontent.com/35657426/70112841-5e800400-1689-11ea-89cd-368aefb22282.png)

**Comment by timy90022** [comment]
Great! 
![image](http://www.pianshen.com/images/633/016239873c8b70c4f3cbede35ce2bff1.png)

**Comment by trungpham2606** [comment]
@howardyclo  can you guys share another checkpoints ( vgg16 resnet101, mobilenet, ...) ?

**Comment by trungpham2606** [comment]
@timy90022 @howardyclo 
Seems it doesnt work well on coco 2014 valid ?
![image](https://user-images.githubusercontent.com/35657426/70122946-35b93800-16a4-11ea-9779-5a0bba852174.png)

**Comment by howardyclo** [comment]
@trungpham2606 Since the following questions are not related to this issue. Would you mind to open the other issues that are related your problem? Thanks! :-)

**Comment by trungpham2606** [comment]
@howardyclo I just created new issue and someone had closed it right away =)))

**Comment by howardyclo** [comment]
@trungpham2606 The issue will be closed as soon as we responsed to your question :-)

**Comment by trungpham2606** [comment]
@howardyclo  yeah yeah I see. I'm also doing this kind of project: unknown objects detection. My pipeline is just compare feature extraction from the backbone net. But my dataset is way smaller than COCO or VOC and it's specific dataset about candies. Maybe with this strategy, it will not be able to handle general case (with a lot of different types of classes). 
I appreciate your prompt support !
